client.js

```diff
@@ -1,6 +1,8 @@
 const Discord = require('discord.js');
 const logger = require('winston');
-const auth = require('./settings.json');
+//settings.test for test server 
+const auth = require('./settings.test.json');
```

Comment:
Really no reason to need a different settings file in code, right? Just swap out the actual file, the idea of a settings file is you don't have to make code changes. But, I guess if that's your workflow, that's fine, since I assume this'll be removed before this merges anyway.

Reply:
It's ignored so shouldn't be a problem.